### PR TITLE
fix(graphing): increase height and width for longer labels for the two axes

### PR DESCRIPTION
**Pull request by CarlaCostea**
https://illuminate.atlassian.net/browse/PD-988

### CODE REVIEW

File: packages/plot/src/utils.js [hunk 1]
```diff
@@ -138,3 +138,29 @@ export const isDomainRangeEqual = (graphProps, nextGraphProps) => {
...
+export const findLongestWord = label => {
+  var longestWord = (label || '').split(' ').sort(function(a, b) {
+    return b.length - a.length;
+  });
+  return longestWord[0].length;
+};
+
+export const amountToIncreaseWidth = longestWord => {
+  switch (true) {
+    case longestWord < 3:
```
Comment (andreeapescar) [inline packages/plot/src/utils.js:151]: I would rather replace this with a formula. (multiplying the nr of characters with a constant)

File: packages/plot/src/utils.js [hunk 2]
```diff
@@ -138,3 +138,29 @@ export const isDomainRangeEqual = (graphProps, nextGraphProps) => {
     isEqual(graphProps.range, nextGraphProps.range)
   );
 };
+
+export const findLongestWord = label => {
+  var longestWord = (label || '').split(' ').sort(function(a, b) {
```
Comment (andreeapescar) [inline packages/plot/src/utils.js:143]: use let

Comment (andreeapescar) [reply packages/plot/src/utils.js:143]: Also, you should take in consideration the fact that this is an html element, not always a String.
(eg: `<i>Long Text</i>,` `<div><strong>A</strong></div>`)

File: packages/graphing/src/axis/axes.jsx [hunk 3]
```diff
@@ -216,7 +223,7 @@ export class RawYAxis extends React.Component {
           <Arrow direction="up" x={0} y={range.max} className={classes.arrow} scale={scale} />
         )}
         {range.axisLabel && (
-          <foreignObject x={scale.x(0) - 50} y={-25} width="100" height="20">
+          <foreignObject x={scale.x(0) - 100} y={-25} width="200" height="20">
```
Comment (andreeapescar) [inline packages/graphing/src/axis/axes.jsx:226]: I think this value should also be dynamically depending on characters number?

File: packages/plot/src/utils.js (hunk 2 above)
Comment (andreeapescar) [reply packages/plot/src/utils.js:143]: replace function (a, b) {...} with (a, b) => {...}

**Review by andreeapescar — CHANGES_REQUESTED** [review]

File: packages/plot/src/utils.js (hunk 2 above)
Comment (CarlaCostea) [reply packages/plot/src/utils.js:143]: ok

File: packages/plot/src/utils.js (hunk 1 above)
Comment (CarlaCostea) [reply packages/plot/src/utils.js:151]: done

File: packages/graphing/src/axis/axes.jsx (hunk 3 above)
Comment (CarlaCostea) [reply packages/graphing/src/axis/axes.jsx:226]: sure

**Review by andreeapescar — APPROVED** [review]

**Pull request merged**